frontend2.0/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
```

Comment:
could you add `npm install`?

Reply:
seems like this README is copied from create react app, could you edit it to explain more of the frontend design?

Reply:
Readme files have been updated.